src/core_seaice/Registry.xml

```diff
@@ -2181,6 +2273,8 @@
 		<package name="pkgColumnTracerPonds" description=""/>
 		<package name="pkgColumnTracerLidThickness" description=""/>
 		<package name="pkgColumnTracerAerosols" description=""/>
+		<package name="pkgColumnTracerSnowDensity" description=""/>
```

Comment:
Add description (I know it seems unfair with nothing else having a description!)